workshops/forms.py

```diff
@@ -109,8 +116,92 @@ class DebriefForm(forms.Form):
     )
 
 
-# forms below are created using a factory instead of class
-# they're used on event edit page
-EventForm = modelform_factory(Event, fields="__all__")
-TaskForm = modelform_factory(Task, fields="__all__",
-                             widgets={'event': HiddenInput})
+class EventForm(forms.ModelForm):
+
+    site = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.SiteLookup,
+        label='Site',
+        required=True,
+        help_text=AUTOCOMPLETE_HELP_TEXT
+    )
+
+    organizer = selectable.AutoCompleteSelectField(
+        lookup_class=lookups.SiteLookup,
+        label='Organizer',
+        required=True,
+        help_text=AUTOCOMPLETE_HELP_TEXT
+    )
```

Comment:
I checked and for fields that don't produce many options (like site, organizer or airport) we should be using [AutoComboboxSelectWidget](http://django-selectable.readthedocs.org/en/latest/widgets.html#autocomboboxselectwidget) - even if it looks quirky on the website.

Then you don't need AUTOCOMPLETE_HELP_TEXT.